.github/workflows/build.yml

```diff
@@ -122,6 +122,11 @@ jobs:
         ./run_example.sh --dry-run application_service run_test
 
         ./run_example.sh --dry-run simple_app_under_keystone
+
+        ./run_example.sh --dry-run simple_app_under_cca
+        ./run_example.sh --dry-run simple_app_under_cca setup
+        ./run_example.sh --dry-run simple_app_under_cca run_test
```

Comment:
Just calling some simple `--dry-run` tests to ensure that we don't regress this script in future for CCA-shim-based sample-app.